boardClass.lua

```diff
@@ -361,10 +361,13 @@ end
 function setUpdateForBoard(board)
     function board:update()
         handleButtonsforBoard(board)
-        local temp = playdate.getCrankTicks(4)
-        if temp ~= 0 then
-            local sign = findSign(temp)
-            incrementSelectedWithCrank("crank",math.floor(sign),board)
+        
+        if not playdate.isCrankDocked() then
```

Comment:
not 100% sure, but 'check if cranked then get ticks'  seems a bit faster than 'get cranks despite state'